ground/gcs/src/plugins/systemhealth/html/SystemConfiguration-Warning-LQG.html

```diff
@@ -0,0 +1,12 @@
+<html>
+  <head>
+    <title></title>
+    <meta content="">
+    <style></style>
+  </head>
+  <body style="color: black">
+    <h1>System Configuration: Warning</h1>
+    <p>At least one LQG-controller based flight mode has been configured, but no valid system identification has been data found. Please rerun the autotune wizard, or if necessary, or not done yet, the complete autotune procedure.</p>
+    <p>Trying to take off this way will not prevent flight, but stabilization will fall through to the PID-controller instead.</p>
```

Comment:
Hm.  I guess this lets you do e.g. lqg/autotune as flight modes and not have to reconfig.. but if you need to connect a computer anyways to apply autotune...  Probably should still mask arming and remove this line.